.terraform.lock.hcl

```diff
@@ -0,0 +1,120 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
According to the[ terraform docs](https://www.terraform.io/language/files/dependency-lock#lock-file-location), this should be included with version control. 
> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.